go/sqltypes/named_result.go

```diff
@@ -110,6 +110,22 @@ func (r RowNamedValues) AsBool(fieldName string, def bool) bool {
 	return def
 }
 
+// ToBytes returns the named field as a byte array
+func (r RowNamedValues) ToBytes(fieldName string) ([]byte, error) {
+	if v, ok := r[fieldName]; ok {
+		return v.ToBytes()
+	}
+	return nil, ErrNoSuchField
+}
+
+// AsBytes returns the named field as a byte array, or default value if nonexistent/error
+func (r RowNamedValues) AsBytes(fieldName string, def []byte) []byte {
```

Comment:
Agree, `def` is confusing. Since used in all other functions, so I went with this rather than just name it differently here or fix it everywhere. But I should have ...